src/App.js

```diff
@@ -72,7 +72,7 @@ class Root extends React.Component {
     // When Ledger device loses connection or the app is closed
     if (this.props.ledgerClosed && !prevProps.ledgerClosed) {
       LOCAL_STORE.lock();
-      this.props.history.push('/wallet_type/');
+      this.props.history.push('/locked/');
```

Comment:
Without this change, a complex navigation rendering takes place, ending up to redirect to `/wallet` path and appearing it didn't changed the view.

On other hand, by changing the redirection to `/locked` path, the returnDefaultComponent function executes a wallet cleanup and and redirect to `/wallet_type` path, our desired destination.

I still have to test by I believe this change also fix this issue:
https://github.com/HathorNetwork/internal-issues/issues/225